exporter/trace/trace_proto_test.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"testing"
+
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"go.opentelemetry.io/otel/sdk/resource"
+
+	"go.opentelemetry.io/otel/api/kv"
+)
+
+func TestInjectLabelsFromResources(t *testing.T) {
+	testcases := []struct {
```

Comment:
I don't see any test cases with conflict between resource and span attributes, e.g.
```go
			export.SpanData{
				Resource: resource.New(kv.String("a", "resource-value")),
				Attributes: []kv.KeyValue{
					kv.String("a", "attribute-value"),
				},
			},

```